### DISCUSSION TITLE
Provide guidance/example on how to use OpenTelemetryResourceAutoConfiguration

### DISCUSSION THREAD

**Discussion by iamnicoj** [open-telemetry/opentelemetry-java#3528] [Q&A]
I am trying to upgrade from previous versions the way I was configuring the Tracer provider using `OpenTelemetrySdkAutoConfiguration.getResource()` with the newer `OpenTelemetryResourceAutoConfiguration` class, but not sure how exactly it should be configured.

**Comment by jkwatson** [comment]
Calling `OpenTelemetryResourceAutoConfiguration.configureResource()` should give you exactly the same Resource that you would have gotten from the old method.
